test/Interop/Cxx/operators/member-inline.swift

```diff
@@ -437,4 +437,16 @@ OperatorsTestSuite.test("HasOperatorCallWithDefaultArg.call") {
   expectEqual(444, res)
 }
 
+OperatorsTestSuite.test("HasStaticOperatorCallBase.call") {
+  let h = HasStaticOperatorCallBase()
+  let res = h(1)
+  expectEqual(43, res)
+}
+
+OperatorsTestSuite.test("HasStaticOperatorCallDerived.call") {
+  let h = HasStaticOperatorCallDerived()
+  let res = h(0)
+  expectEqual(42, res)
+}
+
```

Comment:
Could you please also add a test to `test/Interop/Cxx/class/protocol-conformance-typechecker.swift` that conforms a C++ type with a static `operator()` to a Swift protocol that requires `func callAsFunction`? That would make sure that the Swift lookup table is populated correctly.